baybe/parameters/substance.py

```diff
@@ -58,10 +57,28 @@ class SubstanceParameter(DiscreteParameter):
     """
 
     encoding: SubstanceEncoding = field(
-        default=SubstanceEncoding.MORDRED, converter=SubstanceEncoding
+        default=SubstanceEncoding.MordredFingerprint,
+        converter=lambda x: (
+            # Passed enum
+            x
+            if isinstance(x, SubstanceEncoding)
+            # Passed enum name
+            else (
+                SubstanceEncoding[x]
+                if x in SubstanceEncoding.__members__
+                # Passed enum value
+                else SubstanceEncoding(x)
+            )
+        ),
     )
     # See base class.
 
+    kwargs_fingerprint: dict = field(default={})
+    """Kwargs for fingerprint generator"""
+
+    kwargs_conformer: dict = field(default={})
+    """Kwargs for conformer generator"""
```

Comment:
Several issues here:
* type annotation missing for the dicts
* cannot be `default`, must be `factory=dict` since mutable
* don't like so much that the names start with "kwargs". can we give a more human-readable name?

Reply:
Changed to factory

For the other two comments I agree with @Scienfitz 
Another notation for passing separate kwargs dicts I have seen before in seaborn is e.g. `facet_kws`, although that doesn't change much, except how kwargs is abbreviated.

Reply:
resolving this